__tests__/integration/nestedResources.test.js

```diff
@@ -8,36 +8,64 @@ describe('Expanding a resource property in a property panel', () => {
     return await testUserLogin()
   })
 
-  it('loads up a resource template from the list of loaded templates', async () => {
+  it('loads up the BIBFRAME Instance resource template from the list of loaded templates', async () => {
     expect.assertions(2)
     await pupExpect(page).toClick('a', { text: 'BIBFRAME Instance' })
     await pupExpect(page).toMatch('BIBFRAME Instance')
   })
 
-  it('clicks on one of the property type rows to expand a nested resource', async () => {
+  it('clicks on the Has BIBFRAME Instance property type rows to expand the resource', async () => {
```

Comment:
It looks like this test depends on the previous test being run.  Which means if you do `jest __test__/integration/nestedResources.test.js -t "Instance property"` it would fail. Can you combine the tests so that they can each be run in isolation?

Reply:
yeah, i'm a fan of being able to run tests in isolation.

Reply:
Thanks. I think the setup that @ndushay mentions can address this. This is the kind of input I was looking for as I pick up this stack :-)